docs/docs/intro.md

```diff
@@ -82,4 +82,87 @@ For details, please see [Nebula Discovery](basics/5-discovery.md)
 
 
 ### Kubernetes
-(to be updated)
+
+Suppose you already have a kubernetes cluster. Follow the steps below to deploy nebula in your cluster and access it from your localhost.
+
+* Create a configuration file nebula.yaml.
+
+```shell
+cat > nebula.yaml <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nebula-server
+spec:
+  selector:
+     matchLabels:
+       app: nebula-server
+  template:
+    metadata:
+      labels:
+        app: nebula-server
+    spec:
+      containers:
+        - name: nebula-web
+          image: columns/nebula.web
+          env:
+          - name: NS_ADDR
+            value: "localhost:9190"
+          - name: NODE_PORT
+            value: "8088"
+        - name: nebula-server
+          image: columns/nebula.server
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nebula
+spec:
+  type: NodePort
+  selector:
+    app: nebula-server
+  ports:
+    - port: 9190
+      name: server
+      targetPort: 9190
+    - port: 8088
+      name: web
+      targetPort: 8088
+      nodePort: 30008
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nebula-node
+spec:
+  replicas: 2
+  selector:
+     matchLabels:
+       app: nebula-node
+  template:
+    metadata:
+      labels:
+        app: nebula-node
+    spec:
+      containers:
+        - name: nebula-node
+          image: columns/nebula.node
+          env:
+          - name: NSERVER
+            value: "nebula:9190"
```

Comment:
should this be `nebula-server:9190` or just `nebula:9190`?

Reply:
This is right. See line 119 that the name of service is "nebula"